### PR TITLE
Added dBay in one commit

### PR DESCRIPTION
**Pull request by LuisEGV**
Added image, changed resource.cljs and default_dapps.cljs

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

If you submit PR for issue with bounty then write here Fixes #NN where NN is issue number

*otherwise*

fixes #...

### Summary:

[comment]: # (Summarise the problem and how the pull request solves it)
...

#### Platforms (optional)
- Android
- iOS
- macOS
- Linux
- Windows

#### Areas that maybe impacted (optional)
**Functional**
- 1-1 chats
- public chats
- group chats
- wallet / transactions
- dapps / app browsing
- account recovery
- new account
- user profile updates
- networks
- mailservers
- fleet
- bootnodes

**Non-functional**
- battery performance
- CPU performance / speed of the app
- network consumption

### Steps to test:
- Open Status
- ...
- Step 3, etc.

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/browser/default_dapps.cljs [hunk 1]
```diff
@@ -57,6 +57,10 @@
              :dapp-url    "https://cryptocribs.com"
              :photo-path  "contacts://cryptocribs"
              :description "Travel the globe. Pay in crypto."}
+             {:name        "dBay"
+             :dapp-url    "https://www.dbay.ai/"
```
Comment (jeluard) [inline src/status_im/ui/screens/browser/default_dapps.cljs:61]: Please align key and values or the build fails.

**Review by jeluard — CHANGES_REQUESTED** [review]

**Comment by jeluard** [comment]
@LuisEGV Could you update the map alignment?

**Comment by flexsurfer** [comment]
@rachelhamlin should we close this? cc @andytudhope

**Comment by andytudhope** [comment]
Added in this commit https://github.com/status-im/discover-dapps/commit/c8ed9d99114dd7f533f3d72247c829f112bbaeba

**Pull request closed without merging**